SPECS/python-rpm-spec/python-rpm-spec.spec

```diff
@@ -0,0 +1,45 @@
+Name:           python-rpm-spec
+Version:        0.14.1
+Release:        1%{?dist}
+Summary:        A Python library for parsing RPM spec files.
+
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/bkircher/python-rpm-spec
+Source0:        https://github.com/bkircher/python-rpm-spec/archive/refs/tags/v0.14.1.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
Please run the linter against the spec - this will re-order the tags in correct order and should catch some other inconsistencies (if any are still left). The linter is not perfect - it wraps function macros in brackets breaking argument passing, so it's worth double-checking its changes, but usually it's very helpful.

Reply:
Done
```
azaugg@azaugg-ld77 [ ~/CBL-Mariner/SPECS/python-rpm-spec ]$ rpmlint python-rpm-spec.spec
0 packages and 1 specfiles checked; 0 errors, 0 warnings.
azaugg@azaugg-ld77 [ ~/CBL-Mariner/SPECS/python-rpm-spec ]$
```

Reply:
Sorry for being unclear. I meant the [linter we're using for Mariner specs](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/1126/Spec-linter) - it handles a few more cases we're interested in.